cg-spec/editors_draft.html

```diff
@@ -415,12 +414,124 @@ <h2>Concepts and Abstract Syntax</h2>
 
     <p>Again, this definition is an extension of the notion of <a>RDF dataset</a>, hence it follows that any <a>RDF dataset</a> is also an <a>RDF-star dataset</a>.</p>
 
+    <section class="informative" id="quoted-triples">
+      <h2>Quoted and Asserted Triples</h2>
+
+      <p>
+        A triple in RDF is a tuple of three components: subject, predicate and object. These are
+        collectively known as <dfn data-cite="RDF11-CONCEPTS#section-rdf-graph">RDF terms</dfn>.
```

Comment:
Perhaps we should add "As defined in [[RDF11-CONCEPTS]]" to clarify that this concept is defined elsewhere, without needing to follow the reference.

Reply:
Ok - the link was to the exact section but just the REC works if we don't expect the reader to need to look it up.

Actually, respec was finding the original SPARQL definition for the link when `<a> </a>` is used. Switched to refer to Concepts for the definition of RDF term.